packages/components/pie-webc/package.json

```diff
@@ -228,6 +238,8 @@
     "@justeattakeaway/pie-form-label": "0.14.1",
     "@justeattakeaway/pie-icon-button": "0.28.12",
     "@justeattakeaway/pie-link": "0.18.0",
+    "@justeattakeaway/pie-lottie-player": "0.0.0",
+    "@justeattakeaway/pie-link": "0.18.0",
```

Comment:
pie-link seems to be duplicated

Reply:
😄 Every time I try to solve conflicts in GH UI...